certificate.pem

```diff
@@ -0,0 +1,22 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Could you please move the certificate files (`certificate.pem`, `certs.p12` & `private_key.pem`) to the `resources/` directory?

You could also create a `resources/README.md` file, and mention how those files were created.